CMakeLists.txt

```diff
@@ -10,7 +10,14 @@
 project("ubpf")
 cmake_minimum_required(VERSION 3.16)
 
-if (UBPF_INSTALL_GIT_HOOKS)
+# Only run if not included as a subproject
+if (CMAKE_CURRENT_SOURCE_DIR STREQUAL CMAKE_SOURCE_DIR)
```

Comment:
I don't think this tests if it was included as a subproject.  For example, if you "cd external\ubpf" and then try to cmake from there, this will be true and installing the git hook would still fail since the .git directory doesn't exist.
I think it should instead either test whether .git exists.

Reply:
I get that line 21 now fixes the actual issue.
However I don't see value in lines 14-18, and they certainly don't match the comment in 13 (hence unresolving the conversation).

I'd recommend deleting 13-18 and just having line 21 be the only real change.

Reply:
Simplified the check and only check for the presence of .git/hooks folder.